packages/ethereum-contracts/truffle-config.js

```diff
@@ -129,7 +129,6 @@ function createNetworkDefaultConfiguration(
 const E = (module.exports = {
     plugins: [
         //"truffle-security",
-        "solidity-coverage",
```

Comment:
I removed this because the plugin makes verification fail:
```
$ npx truffle run --network eth-goerli verify ConstantFlowAgreementV1@0x7F3Ba5Ded3C017c5bA0469085AAf0Ba5F7cA42b4
ExtendableError: 
Error: truffle-plugin.json not found in the solidity-coverage plugin package!
```

This error disappears if not enabling the plugin.

As far as I understand, coverage checking is now handled via hardhat and thus can be removed here.
@hellwolf is that correct?
If so, the "coverage" network should likely be removed too from this file.